addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -514,6 +514,25 @@ export function setSelection(
 
     return [anchorNode, anchorOffset, focusNode, focusOffset];
 }
+/**
+ * Get the first and last node of the `nodes` relative to a pre-order
+ * depth-first traversal of the DOM.
+ *
+ * @param {Node[]} nodes
+ */
+export function getFirstAndLastNodes(nodes) {
```

Comment:
This is used only once. Does it really make sense to be its own util ?

Reply:
Any function can be potentially reused. What makes this one special ? What are the cases that you think might arise for which this function would come in handy ?

Reply:
You're right, I can't come with another case.
After re-thinking where I've used it, I think I don't need to use the function there anyway. I've removed it.